src/sdk.js

```diff
@@ -442,6 +443,10 @@ const validateSdkConfig = sdkConfig => {
     throw new Error('baseUrl must be provided');
   }
 
+  if (!sdkConfig.assetCdnBaseUrl) {
```

Comment:
Could this be optional in a similar way than the api-baseurl?
(I guess, it's only us that need it for staging?)

Reply:
Ah, never mind, I read this wrongly...